### DISCUSSION TITLE
`DocumentHandler` seems redundant, is it needed at all?

### DISCUSSION THREAD

**Discussion by ShafSpecs** [ShafSpecs/remix-pwa#94] [Q&A]
By `DocumentHandler`, I mean creating a handler for document caching. I realised that `isDocumentRequest` doesn't seem to do anything at runtime cause the `RemixNavigationHandler` handles document caching and when using `PrecacheHandler`, you don't even need any cache strategies. 

Would moving the logic away from the navigation handler be the solution? Or would it be okay to yank documentHandler from the docs and throw in the bin

**Comment by ShafSpecs** [comment]
cc @dev-afzalansari

**Reply by dev-afzalansari** [reply]
> Would moving the logic away from the navigation handler be the solution? Or would it be okay to yank documentHandler from the docs ? and throw in the bin

We can do both in the next release for now I think it's just an unnecessary piece of code that we can remove in upcoming releases.
